### ISSUE TITLE
ERROR: Could not find latest version. Please double check your HOST_KEY.

### ISSUE THREAD

**Issue by brianmurphy** [cloudflare/CloudFlare-CPanel#168]
The installer doesn't work. I copy and pasted the commands from the install doc. API key copy/pasted from my profile.

Cpanel v70.0.26 on Centos 7.

(Key below sanitized for this public ticket)

./cloudflare.install.sh -v -k XXXXXXXXXXXXXXX -n "XXXXXX LLC"
HOST_KEY        = 'XXXXXXXXXXXXXXXXX'
ORG_NAME        = 'XXXXXXX'
LOCAL_FILE_PATH = ''
Running as root
Starting CloudFlare CPanel Installation...
LATEST_VERSION - ''
ERROR: Could not find latest version. Please double check your HOST_KEY.

Usage: ./cloudflare.install.sh -k HOST_KEY -n 'ORG_NAME' [-f /path/to/local/cpanel.tar.gz ]

**Comment by abdalians** [comment]
getting the same error

**Comment by brianmurphy** [comment]
@abdalians The instructions I used had me pull the key from my account settings.

Later on I found a different login link: https://partners.cloudflare.com/login

The Partners portal link key was different than the main api keys. They worked with the installer.

It would be nice if a partner key was visually different than other cloudflare keys or at minimum return a smarter error message.

Also improve the Readme.md on github to link to the Partners login portal for the key instead of assuming we will know where to find it.

There are failure points throughout the process. Filling some of these points of ambiguity would make the process easier and cut down on support issues like mine.
